.pre-commit-hooks.yaml

```diff
@@ -1,9 +1,11 @@
 ---
 - id: update-changelog
-  name: Update changelog (section Unreleased)
-  entry: python -c 'import subprocess; subprocess.run("cz changelog > /dev/null", shell=True, check=True)'
+  name: update-changelog
+  description: Update changelog (section Unreleased)
+  entry: cz changelog
```

Comment:
at first i think the user shall customize it by themselves. but now I agree with you that setting it into default is better. since it provide a way to keep the user's modification of the changelog.